.changeset/fair-boxes-carry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ultraviolet/ui': minor
+---
+
+update design and add range date
```

Comment:
You can precise the prop that has been added and to what component, for example:

```suggestion
Component `<DateInput />` updates:
- Design update
- New prop available `selectRange` allowing the user to select a range of dates
```